.env.local.example

```
@@ -60,12 +60,21 @@ VORTEX_ACQUIA_SECRET=
+VORTEX_DB_DOWNLOAD_S3_SECRET_KEY=
+#;> DB_DOWNLOAD_SOURCE_S3
+
 #;< DB_DOWNLOAD_SOURCE_CONTAINER_REGISTRY
 # Database image sourced from a container registry.
 
 # The username to log into the container registry.
-VORTEX_CONTAINER_REGISTRY_USER=$DOCKER_USER
+VORTEX_CONTAINER_REGISTRY_USER=
 # The password (token) to log into the container registry.
-VORTEX_CONTAINER_REGISTRY_PASS=$DOCKER_PASS
+VORTEX_CONTAINER_REGISTRY_PASS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VORTEX_CONTAINER_REGISTRY_PASS key should go before the VORTEX_CONTAINER_REGISTRY_USER key

(UnorderedKey)